app/controllers/peeps.rb

```diff
@@ -0,0 +1,32 @@
+class Chitter < Sinatra::Base
+  get '/peeps/index' do
+    @username = current_user ? current_user.username : nil
+    messages = Peep.all(order: [:created_at.desc])
+    @peeps = []
+    messages.each do |message|
+      @peeps << {
+        text: message.text,
+        author: User.get(message.user_id).username,
+        time: message.created_at.strftime("%a, %e %b %Y %H:%M:%S %z")
+    }
```

Comment:
This could all be extracted to your models. Try to keep your controllers skinny.